uproot/write/sink/cursor.py

```diff
@@ -21,6 +21,7 @@ def write_fields(self, sink, format, *args):
         self.index += format.size
 
     def return_fields(self, format, *args):
```

Comment:
All of these methods with `return_` in their names are doing more than just returning a value without side-effects, they're updating the position of the cursor, so it might be better to say `write_` instead of `return_` or some other word that suggests an in-place update.

I know that the cursor doesn't literally write data to a file, which is an argument against using the word `write_`. You have `update_` meaning a different thing below. Maybe the prefix should be `put_`? "Put" connotes a change in state.